src/pytorch_lightning/tuner/tuning.py

```diff
@@ -196,12 +177,15 @@ def scale_batch_size(
         self.trainer.auto_scale_batch_size = False
         return batch_size_finder.optimal_batch_size
 
+    # TODO: update docs
```

Comment:
Will you resolve this TODO in this PR? If not, the todo note should contain more information of what should be updated.